packages/react-app/src/models/Customer.ts

```diff
@@ -0,0 +1,55 @@
+import mongoose from 'mongoose';
```

Comment:
typings and mongoose schema definitions for Customer collection, reflective of @PaulApivat changes